modules/glimpse/chunk/meta.yml

```diff
@@ -0,0 +1,84 @@
+name: "glimpse_chunk"
+description: Defines chunks where to run imputation
+keywords:
+  - chunk
+  - imputation
+tools:
+  - "glimpse":
+      description: "GLIMPSE is a phasing and imputation method for large-scale low-coverage sequencing studies."
+      homepage: "https://odelaneau.github.io/GLIMPSE"
+      documentation: "https://odelaneau.github.io/GLIMPSE/commands.html"
+      tool_dev_url: "https://github.com/odelaneau/GLIMPSE"
+      doi: "10.1038/s41588-020-00756-0"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - input:
+      type: file
+      description: |
+        Target dataset in VCF/BCF format defined at all variable positions.
+        The file could possibly be without GT field (for efficiency reasons a file containing only the positions is recommended).
+      pattern: "*.{vcf,bcf}"
+
+  - region:
+      type: string
+      description: |
+        Target region, usually a full chromosome (e.g. chr20:1000000-2000000 or chr20).
+        For chrX, please treat PAR and non-PAR regions as different choromosome in order to avoid mixing ploidy.
+
+  - window-size:
```

Comment:
You should only add the inputs of the process here. So everything after window-size shouldn't be in here.

Reply:
Okay. If so do I need to specify somewhere all the different non mandatory parameters available for the software ?

Reply:
No that's not necessary as far as I know